### PR TITLE
MQTT Device Tracker: Updated documentation with source type

### PR DESCRIPTION
**Pull request by tenfire**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27595

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/device_tracker.mqtt.markdown [hunk 1]
```diff
@@ -33,6 +34,10 @@ qos:
   description: The QoS level of the topic.
   required: false
   type: integer
+source_type:
+  description: Source type of the device tracker.
+  required: false
```
Comment (frenck) [inline source/_integrations/device_tracker.mqtt.markdown:38]: This has a default, please add that to this block.

File: source/_integrations/device_tracker.mqtt.markdown [hunk 2]
```diff
@@ -22,6 +22,7 @@ device_tracker:
     devices:
       paulus_oneplus: 'location/paulus'
       annetherese_n4: 'location/annetherese'
+    source_type: router
```
Comment (frenck) [inline source/_integrations/device_tracker.mqtt.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @tenfire! 

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/device_tracker.mqtt.markdown (hunk 2 above)
Comment (tenfire) [reply source/_integrations/device_tracker.mqtt.markdown]: I have now pushed these changes

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request closed without merging**